base/ltplain.dtx

```diff
@@ -1718,14 +1729,25 @@
 % \changes{v2.3f}{2021/04/18}{Drop pre-$\varepsilon$-\TeX{} support}
 % \changes{v2.3f}{2021/04/18}
 %                {Add \cs{tracingstacklevels} and \cs{tracinglostchars}=3}
+% \changes{v2.3l}{2025/03/05}{Use 5 as value for \cs{tracinglostchars}
+%  in recent Lua\TeX{}}
 %    \begin{macrocode}
 %<latexrelease>\IncludeInRelease{2021/06/01}{\loggingall}
 %<latexrelease>                 {\tracingstacklevels and \tracinglostchars=3}%
 %<*2ekernel|latexrelease>
 \edef\loggingall{%
   \tracingstats\tw@
   \tracingpages\@ne
-  \tracinglostchars\thr@@
+  \tracinglostchars
```

Comment:
as discussed in the comments, 3/5 is consistent with the change making it 3 back in 2021 but I think that was in error and over-compensenation for the fact that we increased the default from 1 to 2 at that time.

I don't think it's expected that `\tracingall` makes non-error conditions into errors, so I would make this 2 and 4 or (more simply) remove the setting from here and from `\tracingnone` so the generic tracing on/off commands leave this at its default level.

Reply:
If you want to set to 2 all of the time, does it even need to be in `\tracingall`, etc.? The standard setting is now 2 anyway. (Or are we worrying about places that set to zero/one not being reset?)

Reply:
I agree \tracingall should not make it an error (it should simply leave the setting alone) but the recommendation for users in ltenews is to set it to 3 (to always make it an error) and that sensible and thus the new recommendation for LuaTeX would need to be to set it to 5 not 4.

Reply:
@josephwright  "does it even need to bein `tracingall`" N it doesn't, that's what @FrankMittelbach  and I suggested: "or (more simply) remove the setting from here "

Reply:
(Do we need rollback for that? I'm guessing no)

Reply:
It doesn't need rollback but the whole setting of \tracinglostchars shoulw be removed including the next 9 lines setting it to 3 or 5. And the same with \tracingnone. It should stay transparent to whatever LaTeX has set or the user has changed to.

If there is a setting it should first save the current value (to return to that in \tracingnone) and then set it to 2/4 but I think it is over-engineering and we should take out the set as suggested by @davidcarlisle